### PR TITLE
[v2] Add test coverage for existing code and small refactoring

### PR DESCRIPTION
**Pull request by alloy**
Closes https://artsyproduct.atlassian.net/browse/PLATFORM-1444

### CODE REVIEW

File: src/schema/v2/index.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ import {
   GravityIDFields,
```
Comment (zephraph) [inline src/schema/v2/index.ts]: How would you feel about renaming this to `v2.ts` and having the index file just `export * from './v2'`?

Comment (alloy) [reply src/schema/v2/index.ts]: I’m planning on moving things out with better names, but yeah the index will end up being just an export 👍

**Comment by mzikherman** [comment]
I'm loving the code refactors and specs! I was trying to get the behavior of the transforms clear in my head reading (first from `master`, and then this branch). The code is readable and you could infer the general behavior of each transform pretty easily. Happy that there's specs too which explicitly spell it out.

- __id becomes id

For Gravity types
-----------------
  - id becomes gravityID
  - _id becomes internalID

For other types
---------------
  - id becomes internalID

with some exceptions.

And so generally speaking- adding new fields or types should be done as usual (and in V1), and they will automatically show up (properly transformed) in V2. And then a client running latest Relay w/o the `__id` fork can safely use V2?

**Comment by alloy** [comment]
Correct 👍 

At some point we’ll want to discuss if we should invert it and work on v2 and transform back to v1, but we’re not there yet at all.

**Pull request merged**